web-marketplace/src/components/organisms/EditProfileForm/EditProfileForm.ActionBar.tsx

```diff
@@ -1,20 +1,61 @@
 import { useFormState } from 'react-hook-form';
 import { Box } from '@mui/material';
+import { makeStyles } from 'tss-react/mui';
 
 import ContainedButton from 'web-components/src/components/buttons/ContainedButton';
+import { SaveIcon } from 'web-components/src/components/icons/SaveIcon';
 import { StickyBar } from 'web-components/src/components/sticky-bar/StickyBar';
+import { Theme } from 'web-components/src/theme/muiTheme';
 
 import { SAVE } from './EditProfileForm.constants';
 
-export const EditProfileFormActionBar = () => {
+const useStyles = makeStyles()((theme: Theme) => ({
```

Comment:
since this is copying the style directly from `EditProjectPageFooter`, as we discussed in our 1-1, it'd be better to create a new small component for this save button I guess

Reply:
Happy to do that. Sorry I misunderstood this in our conversation.